coos.html

```diff
@@ -511,49 +523,51 @@ <h3>Base GSBPM model</h3>
 
 				<p>The GAMSO activity areas are not really activities by themselves, but rather “boxes” used to classify the activities. This makes them instances of <code>skos:Concept</code> to represent this “taxonomic” nature, or more precisely of a specific <code>ActivityArea</code> class, which is a sub-class of <code>skos:Concept</code>.</p>
-
-				<pre class="example nohighlight" title="Examples of GAMSO activity area and activity">
-				&lt;http://id.unece.org/activities/activityArea/1&gt; a coos:ActivityArea ;
-					skos:notation "1" ;
-					skos:prefLabel "Strategy and Leadership"@en .
-
-				&lt;http://id.unece.org/activities/activity/2.3&gt; a coos:Activity ;
-					skos:notation "2.3" ;
-					skos:prefLabel "Monitor Capability Improvements"@en .
-				</pre>
-
+				
 				<p>In addition, COOS defines the <code>OverarchingActivity</code> class, which represents a supporting activity that applies to all production phases, e.g. metadata management, data management, statistical methodology management, etc. These activities correspond to the overarching processes in the GSBPM and the statistical activities in Corporate Support in GAMSO. These activities support the execution of phases and sub-processes effectively. When harmonized and standardized, overarching activities enable the efficient execution of the entire statistical production process.</p>
+				
+				<p>Three individuals of <code>ActivityCategory</code> are define in the ontology to model the types of overarching activities provided by GSBPM</p>
 
 				<pre class="example nohighlight" title="Examples of GSBPM overarching activity">
-				&lt;http://id.unece.org/activities/overarchingActivity/0.2&gt; a coos:OverarchingActivity ;
+				&lt;http://id.unece.org/activities/overarchingActivity/0.2&gt; a coos:ActivityCategory ;
 					skos:notation "0.2"	;
 					skos:prefLabel "Metadata Management"@en .
 				</pre>
 
-				<p>As with GSBPM, COOS defines individuals for each GAMSO activity and activity area, and the model itself is a <code>skos:ConceptScheme</code> with all four activity areas having <code>skos:broader</code> links to it and <code>skos:narrower</code> links to the activities they contain. At the top of the scheme, an instance of <code>Activity</code> represents the whole activity of statistical organizations and is named <code>officialStatistics</code>.</p>
+				<p>As with GSBPM, COOS defines individuals for each GAMSO activity and activity area, and the model itself is a <code>skos:ConceptScheme</code> with all four activity areas having <code>skos:broader</code> links to it and <code>skos:narrower</code> links to the categories of activity they contain. At the top of the scheme, an instance of <code>ActivityCategory</code> represents the whole activity of statistical organizations and is named <code>officialStatistics</code>.</p>
```

Comment:
Shorter sentences are needed.

> with all four activity areas having <code>skos:broader</code> links to it

If this is really the case, then this is wrong. Root concepts do not have skos:broader to the ConceptScheme. They have skos:topConceptOf.

A nice indented list could show the scheme content.

Reply:
the individual officialStatistics is the one with the link with the four activity areas. the text has been changed to make it clear